src/components/Cropper/index.js

```diff
@@ -161,7 +160,6 @@ class Cropper extends React.Component {
               <input
                 key={nonce}
                 type="file"
-                className="upload"
```

Comment:
Dropped `"upload"` here in favor of the `{inputStyle}` - let me know if this isn't correct

Reply:
@codeluggage this should remain as it is. This is a hidden field and required by cropper.

Reply:
Oh, so it can handle two references to `className` here? If it should have both, perhaps we should do `className={`upload ${inputStyle}}`?

Reply:
this is correct. thanks :)